service/health/response.go

```diff
@@ -17,8 +17,11 @@ type ClusterStatus struct {
 
 // NodeStatus holds data about the health/status of a node.
 type NodeStatus struct {
-	Name     string `json:"name"`
-	Ready    bool   `json:"ready"`
-	IP       string `json:"ip"`
-	Hostname string `json:"hostname"`
+	Name         string `json:"name"`
+	Ready        bool   `json:"ready"`
+	IP           string `json:"ip"`
+	Hostname     string `json:"hostname"`
+	InstanceType string `json:"instance_type"`
+	CPUCount     int64  `json:"cpu_count"`
+	Memory       string `json:"memory"`
```

Comment:
Arguable if string is the correct type here IMO.

Reply:
The underlying object is a resource.Quantity so we get nicely-formatted strings with standard units that way. Alternatively we can choose a unit for the field (bytes? KB?) and wrangle the number into that unit to report as an int (which is nicer for machines to read I realize) but from reading the doc on Quantity that seems to be exactly what the k8s devs wanted to avoid

Reply:
We might want to do some unit wrangling in the frontend to change display and so on. I would think memory in KB is sufficient - maybe even just GB?

Reply:
I went with bytes as it is most specific and easy to report and then a caller can do whatever they want with the value